sdk/eventhub/eventhubs-checkpointstore-blob/README.md

```diff
@@ -1,19 +1,138 @@
+
+  If a reader disconnects from a partition, when it reconnects it begins reading at the checkpoint that was previously submitted by the last reader of that partition in that consumer group. When the reader connects, it passes the offset to the event hub to specify the location at which to start reading. In this way, you can use checkpointing to both mark events as "complete" by downstream applications, and to provide resiliency if a failover between readers running on different machines occurs. It is possible to return to older data by specifying a lower offset from this checkpointing process. Through this mechanism, checkpointing enables both failover resiliency and event stream replay..
+
+- Event Processor based application consists of one or more instances of `EventProcessor` which have been
+configured to consume events from the same Event Hub and consumer group. They balance the
+workload across different instances by distributing the partitions to be processed among themselves.
+They also allow the user to track progress when events are processed using checkpoints.
+ 
+  A checkpoint is meant to represent the last successfully processed event by the user from a particular
+  partition of a consumer group in an Event Hub instance.
+ 
+  A Partition Manager is a class that implements key methods required by the Event Processor to balance load and update checkpoints.
 
 ## Examples
 
+- [Create a Partition Manager using Azure Blob Storage](#create-a-partition-manager-using-azure-blob-storage)
+- [Checkpoint events using Azure Blob storage](#checkpoint-events-using-azure-blob-storage)
+
+### Create a Partition Manager using Azure Blob Storage
+
+ you can use the below code snippet to create a Partition Manager
+
+```javascript
+import { ContainerClient } from "@azure/storage-blob",
+import { BlobPartitionManager } from "@azure/eventhubs-checkpointstore-blob"
+
+const containerClient = new ContainerClient("storage-connection-string", "container-name");
+await containerClient.create(); // This can be skipped if the container already exists
+
+const partitionManager =  new BlobPartitionManager(containerClient);
+```
+
+### Checkpoint events using Azure Blob storage
+
+To checkpoint events received using an EventProcessor to Azure Blob Storage, you will need to pass an instance of the Partition Manager to the Event Processor along with the code to call the `updateCheckpoint()` method.
+
+In this example, we will use a `SamplePartitionProcessor` that extends the `PartitionProcessor` class in order to checkpoint the last event in the batch.
```

Comment:
This example says we'll checkpoint the last event in the batch, but we aren't actually showing the code to do that. Can you include something like 
```typescript
processEvents(events, partitionContext) {
  if (events.length) {
    /* custom logic for processing events, then checkpoint*/
    await partitionContext.updateCheckpoint(events[events.length - 1]);
  }
}
```